src/internal/connection-provider-routing.js

```diff
@@ -37,7 +37,7 @@ const UNAUTHORIZED_ERROR_CODE = 'Neo.ClientError.Security.Unauthorized'
 const DATABASE_NOT_FOUND_ERROR_CODE =
   'Neo.ClientError.Database.DatabaseNotFound'
 const SYSTEM_DB_NAME = 'system'
-const DEFAULT_DB_NAME = ''
+const DEFAULT_DB_NAME = null
```

Comment:
Was this discovered with new default db test? I didn't see any failure

Reply:
I discovered it when I test against clusters using testkit locally.